### DISCUSSION TITLE
FP Error messages:  how to copy and paste them into nodes or other sw?

### DISCUSSION THREAD

**Discussion by Illegit** [freeplane/freeplane#1159] [Help]
Hi, I ask if anybody knows how to collect the error messages in order to copy and paste them into maps and/or other sw?
Thanks

**Comment by sifran-github** [comment]
you can find error log files in tools->Open user directory, and go into the logs directory.

**Comment by Illegit** [comment]
Hi, sorry I wasn't so precise, my question is referring to the error pop-ups that FP generates when you run a script: see in the image
![Screenshot 2023-05-21 144520](https://github.com/freeplane/freeplane/assets/62674157/ed56ee17-9917-49eb-ac94-c49369bb6f22)
